src/mplhep/styles/lhcb2.py

```diff
@@ -0,0 +1,115 @@
+"""LHCb-like plot style 2
+
+This style resembles the LHCb plotting style; however it is an approximation and
+not an official style.
+
+In order to optimally use the style, the
+the following lines should be used in your python script:
+  plt.xlabel(..., ha='center', y=1)
+  plt.ylabel(..., ha='right', x=1, y=1)
+  plt.legend(loc='best')
+  plt.minorticks_on()
+
+Contributed by Jonas Eschle <Jonas.Eschle@cern.ch>
+based on the works of Kevin Dungs, Tim Head, Thomas Schietinger,
+                      Andrew Powell, Chris Parkes, Elena Graverini
+                      and Niels Tuning
+"""
+
+import matplotlib as mpl
+from cycler import cycler
+
+colors = [
+    "#0078FF",
+    "#FF6600",
+    "#0AAFB6",
+    "#FF3333",
+    "#0000FF",
+    "#00CC00",
+    "#BF8040",
+    "#FF33CC",
+    "#FF7733",
+    "#BFD1D4",
+]
+markers = ["o", "s", "D", "^", "v", "<", ">", "P", "X", "*"]
+
+LHCb2 = {
+    # Plot properties
+    "axes.labelsize": 32,
+    "axes.linewidth": 2,
+    "axes.facecolor": "white",
+    # Custom colors
+    "axes.prop_cycle": cycler("color", colors),
+    # + cycler("marker", markers)  # TODO: markers?
+    "axes.formatter.min_exponent": 3,
+    # Figure properties
+    "figure.figsize": (8, 8),
+    "figure.dpi": 2500,
```

Comment:
This is a very strange choice of dpi, leading to a huge file.

Reply:
yep, true, I reduced that to 300